libs/cardano-cddl/app/Main.hs

```diff
@@ -0,0 +1,34 @@
+{-# LANGUAGE ImportQualifiedPost #-}
```

Comment:
Let's not use this extension for consistency sake. If we are to switch to this style, then we would do this for the whole codebase, not just individual modules.
```suggestion
```